index.js

```diff
@@ -1,61 +1,104 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable radix */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Linting is a great tool to help you write clean code and improve the quality of your code so it is not always a good idea to disable them, in your case, it can be better to extract one class with its logic  into a different file and import it so as to use it